lib/archethic/transaction_chain/transaction/proof_of_validation.ex

```diff
@@ -0,0 +1,178 @@
+defmodule Archethic.TransactionChain.Transaction.ProofOfValidation do
+  @moduledoc """
+  Handle the Proof Of Validation signatures containing
+  - The aggregated signatures
+  - Nodes bitmask (bitmask of nodes used for the signatures)
+
+  Proof of Validation aggregate all valid Cross Validation Stamp signatures
+  It require a threshold of cross stamp without error based on the expected number
+  of validations nodes
+  """
+
+  alias Archethic.Crypto
+
+  alias Archethic.Election
+  alias Archethic.Election.StorageConstraints
+
+  alias Archethic.P2P.Node
+
+  alias Archethic.TransactionChain.Transaction.CrossValidationStamp
+  alias Archethic.TransactionChain.Transaction.ValidationStamp
+
+  alias Archethic.Utils
+
+  alias __MODULE__.SortedNode
+
+  @enforce_keys [:signature, :nodes_bitmask]
+  defstruct [:signature, :nodes_bitmask, version: 1]
+
+  @type t :: %__MODULE__{
+          version: non_neg_integer(),
+          signature: binary(),
+          nodes_bitmask: bitstring()
+        }
+
+  defmodule SortedNode do
```

Comment:
Suggestion: you could make plural for the name of this struct.

Reply:
Agreed and it was already renamed on last commit as `ElectedNodes`